spec/views/gardens/_area.html.haml_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
```

Comment:
Unfortunately, view specs are deprecated! Sorry this wasn't more obvious. There's a PR for adding notice at the top of each spec so it will be more clear not to write these. Would you mind removing this spec and replacing it with a feature spec? 

You can probably integrate a test for these in the existing garden spec: `spec/features/gardens_spec.rb`. And probably no need to check every case - just probably no area and one other.

Additionally, we are using the `expect` syntax. So please replace lines like `response.should match(/m\n<sup>2<\/sup>/i)` with something like: `expect(response).to match(/m\n<sup>2<\/sup>/i)`